imports/client/components/HuntProfileListPage.tsx

```diff
@@ -18,15 +18,16 @@ const HuntProfileListPage = () => {
 
   const usersLoading = useSubscribe('huntMembers', huntId);
   const userRolesLoading = useSubscribe('huntUserInfo', huntId);
-  const profilesLoading = useSubscribe('profiles');
```

Comment:
Same thing here as in `AllProfileListPage` -- we're missing fields that `ProfileList` would benefit from having.